spec/docs/joinconditions.md

```diff
@@ -43,7 +43,7 @@ has exactly one value for each of the following two properties:
 * a [child map]() (`rml:childMap`),
   whose value is an [Expression Map]() (`rml:ExpressionMap`), which
   MUST include references that exists in the [Logical Source]()
-  of the [Parent Triples Map]() that contains the [Referencing Object Map]()
+  of the [Triples Map]() that contains the [Referencing Object Map]()
```

Comment:
This correction makes sense, but the whole sentence doesn't IMO.

I don't think we can say that references _exists_ in a logical source. And I also don't think it's a MUST. 

I think the sentence should be something along the lines of:

> * a child map property (`rr:childMap`) whose value is a child map (`rml:ChildMap`).

And then there should be explanation about what a `rml:ChildMap` is.
Whether or not the child map expression resolves or not is not really a concern for the spec IMO.